context.go

```diff
@@ -0,0 +1,22 @@
+package slogx
+
+import (
+	"context"
+
+	"golang.org/x/exp/slog"
```

Comment:
I believe we've decided to use log/slog and go 1.21 here.
This will require changes in go.mod and CI&CD workflow.